_layouts/index.liquid

```diff
@@ -65,6 +65,9 @@ const apiKeyAirtable = '{{ env.AIRTABLE_API_KEY }}';
 const baseId = '{{ env.AIRTABLE_BASE_ID }}';
 const table = '{{ env.AIRTABLE_TABLE_ID }}';
 
+console.log('Base ID:', baseId);
```

Comment:
Do we need this? ✂️

Reply:
I want to know if these variables are there now that we are doing the redirection, but if you have another way to check that I'll delete it :)

Reply:
Since we are not minifying the JS in the page it should be easy to inspect them with the debugger setting up a break point in the code. It's good exercise if you haven't used it ;)

Reply:
Thanks!  had completely forgotten about its existence 🌚